### PR TITLE
Enhanced Readme

### PR DESCRIPTION
**Pull request by eagleanurag**
More attractive Readme effort by me...

### CODE REVIEW

**Comment by thias15** [comment]
The shields are very nice. The new image for the youtube video is also ok. However, the gif banner from the website is not good. It's too big and does not look good especially on mobile phones. Also the duplication of titles in banner and text is not nice. The Readme should be quick to load and easy to read. Please update the PR.

**Comment by thias15** [comment]
The animated banner could maybe be at the bottom. If it takes some time to load for ppl with slow connection it doesn't matter.

**Comment by eagleanurag** [comment]
@thias15 I Updated the PR as guided. I hope now it will be better in looks from the previous one....

File: README.md [hunk 1]
```diff
@@ -28,11 +35,17 @@ Please cite our [paper](docs/paper/OpenBot.pdf) if you use OpenBot.
...
+<a href="https://www.youtube.com/watch?v=qc8hFLyWDOM" >
+  <img align="center" width="500" alt="youtube video" src="docs/images/youtube.jpg" />
+</a>
 
 ## Contact
 
 - Join our [Slack](https://join.slack.com/t/openbot-community/shared_invite/zt-hpso8cfl-JO9OVhVMdUWvR4vDXwcMGA) channel to connect with the OpenBot community.
 - Contact us via [Email](mailto:openbot.team@gmail.com)
+
+<a href="https://www.openbot.org//" target="_blank">
```
Comment (francisduvivier) [inline README.md:49]: Is the double slash here intentional?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
```
Comment (francisduvivier) [inline README.md:1]: Is the double slash here intentional (at the end of "openbot.org//")?

**Pull request closed without merging**